code/game/objects/items/weapons/clown_items.dm

```diff
@@ -165,6 +167,39 @@
 					continue
 			M.emote("flip")
 
+/obj/item/weapon/bikehorn/saxophone
+	name = "saxophone"
+	desc = "NEVER GONNA DANCE AGAIN, GUILTY FEET HAVE GOT NO RHYTHM!"
+	icon = 'icons/obj/musician.dmi'

```

Comment:
To make sure we can easily merge upstream changes from /tg/station back into our repo you need to place new content in the correct folders, this will also make it easier to rebase if we ever need to do so in the future.

- All new code should go into `/code/modules/hippie`
- All new icons should go into `/icons/hippie`
- All new sounds should go into `/sounds /hippie`